e2e/specs/modal.js

```diff
@@ -1,6 +1,6 @@
 const bsStatus = require('../browserstack-send-status')
 
-const baseURL = 'http://localhost:8080/modal'
+const baseURL = 'http://localhost:8080/modal/'
```

Comment:
https://github.com/vitejs/vite/issues/1831
Only nested/ with an ending slash will be considered a implicit index. Without the slash it's an SPA fallback.
@posva

Reply:
This is okay, thanks for the change 👍